packages/swirl-components/src/components/swirl-tag/swirl-tag.css

```diff
@@ -29,62 +29,146 @@
+  }
+}
+
 .tag--intent-info {
   color: var(--s-text-info);
   background-color: var(--s-surface-info-subdued);
 
   & .tag__removal-button {
-    color: var(--icon-info);
+    color: var(--s-icon-info);
   }
 
-  &.tag--bordered {
+  &.tag--variant-outline {
     border-color: var(--s-border-info);
   }
+
+  &.tag--variant-strong {
+    color: var(--s-text-on-action-primary);
+    background-color: var(--s-surface-info-default);
+
+    & .tag__removal-button {
+      color: var(--s-icon-on-action-primary);
+    }
+  }
 }
 
 .tag--intent-critical {
   color: var(--s-text-critical);
   background-color: var(--s-surface-critical-subdued);
 
   & .tag__removal-button {
-    color: var(--icon-critical);
+    color: var(--s-icon-critical);
   }
 
-  &.tag--bordered {
+  &.tag--variant-outline {
     border-color: var(--s-border-critical);
   }
+
+  &.tag--variant-strong {
+    color: var(--s-text-on-action-primary);
+    background-color: var(--s-surface-critical-default);
+
+    & .tag__removal-button {
+      color: var(--s-icon-on-action-primary);
+    }
+  }
 }
 
 .tag--intent-warning {
   color: var(--s-text-warning);
   background-color: var(--s-surface-warning-subdued);
 
   & .tag__removal-button {
-    color: var(--icon-warning);
+    color: var(--s-icon-warning);
   }
 
-  &.tag--bordered {
+  &.tag--variant-outline {
     border-color: var(--s-border-warning);
   }
+
+  &.tag--variant-strong {
+    color: var(--s-text-on-action-primary);
+    background-color: var(--s-surface-warning-default);
+
+    & .tag__removal-button {
+      color: var(--s-icon-on-action-primary);
+    }
+  }
 }
 
 .tag--intent-success {
   color: var(--s-text-success);
   background-color: var(--s-surface-success-subdued);
 
   & .tag__removal-button {
-    color: var(--icon-success);
+    color: var(--s-icon-success);
   }
 
-  &.tag--bordered {
+  &.tag--variant-outline {
     border-color: var(--s-border-success);
   }
+
+  &.tag--variant-strong {
+    color: var(--s-text-on-action-primary);
+    background-color: var(--s-surface-success-default);
+
+    & .tag__removal-button {
+      color: var(--s-icon-on-action-primary);
+    }
+  }
+}
+
+.tag--intent-special {
+  color: var(--s-decorative-grape-text);
+  background-color: var(--s-decorative-grape-surface-subdued);
+
+  & .tag__removal-button {
+    color: var(--s-decorative-grape-text);
+  }
+
+  &.tag--variant-outline {
+    border-color: var(--s-decorative-grape-text);
+  }
+
+  &.tag--variant-strong {
+    color: var(--s-text-on-action-primary);
+    background-color: var(--s-decorative-grape-surface);
+
+    & .tag__removal-button {
+      color: var(--s-icon-on-action-primary);
+    }
+  }
+}
+
+.tag--intent-translucent {
+  color: var(--s-text-on-image);
+  background-color: var(--s-surface-on-image-default);
+
+  & .tag__removal-button {
+    color: var(--s-icon-on-image);
+  }
+
+  &.tag--variant-outline {
+    border-color: var(--s-text-on-image);
+  }
+}
+
+.tag__icon {
```

Comment:
Issue: The position is off. I think it's missing a `margin-left: calc(-1 * var(--s-space-4))`
<img width="94" alt="Bildschirmfoto 2024-07-17 um 11 35 57" src="https://github.com/user-attachments/assets/163cb244-c64b-49c8-b992-60392b01f5ca">
vs.
<img width="107" alt="Bildschirmfoto 2024-07-17 um 11 35 53" src="https://github.com/user-attachments/assets/76a1e989-5417-41f2-a2f9-691df79ab103">

Reply:
Right, I missed that.